study.md

```diff
@@ -66,15 +68,15 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- No because the_sun is an independent individual variable -->
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- No because changing nearest_star is not affected by changes made to the_sun, changed values are not stored for nearest_star, it keeps the previous value assigned.-->
```

Comment:
`the_sun` and `nearest_star` reference the same object in memory so modifying one will affect the other.